bus-mapping/src/evm/opcodes/create.rs

```diff
@@ -135,6 +135,9 @@ impl<const IS_CREATE2: bool> Opcode for Create<IS_CREATE2> {
         // ErrNonceUintOverflow occurred.
         let is_precheck_ok =
             depth < 1025 && caller_balance >= callee.value && caller_nonce < u64::MAX;
+        if !is_precheck_ok {
```

Comment:
Should we only use this for ccc?
Seems `test_create_insufficient_balance` and `test_create_nonce_uint_overflow` are failed (since we handle these errors in create).